### ISSUE TITLE
"Override Default Resolution" setting ignored

### ISSUE THREAD

**Issue by jmans25** [wackerl91/script.luna#144]
I cannot get the "Override Default Resolution" setting to work. It just ignores whatever I have inputted, and runs at 1920x1080. I have enabled the setting, and set the resolution to 1280x720. I have also tried the "enable custom resolution" option and it doesn't work either.

Am I doing something wrong?

Information:
- Luna Version: 0.7.0~rc7
- nVidia GeForce Experience Version: 3.4.0.70
- Moonlight Embedded Version: 2.2.2 
- OSMC Version: 4.4.27-7-osmc

**Expected Behaviour:**
"Override Default Resolution" overrides the auto resolution detection

**Actual Behaviour:**
The "Override Default Resolution" setting is ignored.

Thanks, any help would be appreciated!

**Comment by wackerl91** [comment]
No, you're not doing anything wrong. Those settings are supposed to work just as you described using them; if they don't then it's most likely a bug.
The only confusing thing is that [moonlight-embedded's Wiki](https://github.com/irtimmer/moonlight-embedded/wiki/Usage) states that the default resolution is 1280x720, but I don't know if this has been changed and the Wiki hasn't been updated accordingly?

Could you have a look at the `luna.conf` file and paste its contents here (you may strip any personal information like host ip if it's still in there) when "Override Default Resolution" is enabled and set to 1280x720? In the config file there should be width / height attributes under the "general" section.
It's usually located at `~/.kodi/userdata/addon_data/script.luna/.storage/luna.conf`. 
Note that the file is only updated before a game is launched to prevent unnecessary disk access, So you'll need to change your settings, launch a game and when the stream starts the config file should have changed and reflect your add-on settings.

For additional reference, [here](https://github.com/wackerl91/luna/blob/develop/resources/lib/util/confighelper.py#L103) is the section of the code that handles writing the add-on settings to an external file that can be read by moonlight.

I'll have a look at this myself as well as soon as I can. Thanks for letting me know!

**Comment by jmans25** [comment]
Thank you for your reply dev! 

```bash
$ cat ~/.kodi/userdata/addon_data/script.luna/.storage/luna.conf
[General]
binpath = /usr/bin/moonlight
fps = 30
bitrate = -1
packetsize = 1024
sops = True
remote = True
localaudio = False
surround = False
width = 1280
height = 720
audio = hw:0,0

[Input 0]
mapping = /home/osmc/GamePad.map
input = /dev/input/event0
```

***Update***
I forgot to mention that I tried moonlight from the command line via SSH manually 
`moonlight -720 -60fps`
And it worked properly. I did not try to specify `-height` and `-width` manually, however. 
With that command, steam and Rocket League both run correctly @720p
With luna, Steam runs correctly @720p but RL doesn't (it runs at 1080). 
I haven't tried any other game at this time, as I am new to steam and don't own much more :(

Hope the additional info helps.

**Comment by wackerl91** [comment]
Switching the resolution between Steam Big Picture and the actual game sounds fishy.
From the additional information you provided it might be worth a shot to add -720 or -1080 keys to the config file if the respective resolution is enabled in Luna's settings (instead of individual height / width values).
I'll investigate this further (probably on the weekend) and try to find a fix for it.

Thanks for your taking your time to provide additional details!
